Code/Source/RobotImporter/URDF/CollidersMaker.cpp

```diff
@@ -0,0 +1,309 @@
+
+            if (result.GetResult() != AZ::SceneAPI::Events::ProcessingResult::Success)
+            {
+                AZ_Error("CollisionMaker", false, "Scene updated");
+                return;
+            }
+
+            auto assetInfoFilePath = azMeshPath;
+            assetInfoFilePath += ".assetinfo";
+            AZ_Printf("CollisionMaker", "Saving collider manifest to %s", assetInfoFilePath.c_str());
+            scene->GetManifest().SaveToFile(assetInfoFilePath.c_str());
+
+            bool assetFound = false;
+            AZ::Data::AssetInfo assetInfo;
+            AZStd::string watchDir;
+            AzToolsFramework::AssetSystemRequestBus::BroadcastResult(
+                assetFound,
+                &AzToolsFramework::AssetSystem::AssetSystemRequest::GetSourceInfoBySourcePath,
+                azMeshPath.c_str(),
+                assetInfo,
+                watchDir);
+
+            // Set export method to convex mesh
+            auto readOutcome = AZ::JsonSerializationUtils::ReadJsonFile(assetInfoFilePath.c_str());
+            if (!readOutcome.IsSuccess())
+            {
+                AZ_Error("CollisionMaker", false, "Could not read %s", assetInfoFilePath.c_str());
+                return;
+            }
+            rapidjson::Document assetInfoJson = readOutcome.TakeValue();
+            auto manifestObject = assetInfoJson.GetObject();
+            auto valuesIterator = manifestObject.FindMember("values");
+            if (valuesIterator == manifestObject.MemberEnd())
+            {
+                AZ_Error("CollisionMaker", false, "Invalid json file: %s", assetInfoFilePath.c_str());
+                return;
+            }
+            auto valuesArray = valuesIterator->value.GetArray();
+            for (auto& value : valuesArray)
+            {
+                auto object = value.GetObject();
+
+                auto physXMeshGroupIterator = object.FindMember("$type");
+                AZStd::basic_string<char> physXMeshGroupType = "{5B03C8E6-8CEE-4DA0-A7FA-CD88689DD45B} MeshGroup";
+
+                if (AZ::StringFunc::Equal(physXMeshGroupIterator->value.GetString(), physXMeshGroupType))
+                {
+                    value.AddMember(rapidjson::StringRef("export method"), rapidjson::StringRef("1"), assetInfoJson.GetAllocator());
+                }
+            }
+
+            auto saveOutcome = AZ::JsonSerializationUtils::WriteJsonFile(assetInfoJson, assetInfoFilePath.c_str());
+            if (!saveOutcome.IsSuccess())
+            {
+                AZ_Error("CollisionMaker", false, "Could not save %s", assetInfoFilePath.c_str());
+                return;
+            }
+
+            // Add asset to expected assets list
+            if (assetFound)
+            {
+                auto pxmeshAssetPath = AZStd::string(assetInfo.m_relativePath.c_str());
+                AzFramework::StringFunc::Path::ReplaceExtension(pxmeshAssetPath, ".pxmesh");
```

Comment:
AZ::IO::FixedMaxPath can be used for path manipulation.
```suggestion
                auto pxmeshAssetPath = AZ::IO::Path(assetInfo.m_relativePath).ReplaceExtension("pxmesh");
```